### ISSUE TITLE
support shifting multiple values from the list

### ISSUE THREAD

**Issue by antulik** [nateware/redis-objects#192]
This command would be handy for batch processing, this is my simple version of it

``` ruby
def shift_many count
    return [] if count <= 0

    results = redis.multi do
      redis.lrange(key, 0, count - 1)
      redis.ltrim(key, count, -1)
    end

    results.first.map { |v| unmarshal(v) }
  end
```

**Comment by TiSer** [comment]
Yes, add this one, please.

**Comment by nateware** [comment]
Just released 1.3.0 which includes this - thanks for your patience!
